lib/topological_inventory/core/logging.rb

```diff
@@ -0,0 +1,19 @@
+require "manageiq/loggers"
+
+module TopologicalInventory
+  module Core
+    class << self
+      attr_writer :logger
+    end
+
+    def self.logger
+      @logger ||= ManageIQ::Loggers::Container.new
```

Comment:
@agrare I recently put a PR into a different repo dealing with logger in test mode.  I wonder if it's better for this line to just be 

```ruby
@logger ||= Rails.env.test? ? Logger.new("/dev/null") : ManageIQ::Loggers::Container.new
```

Reply:
Additionally, I wonder if we should do this in the logger helper from manageiq-api-common, otherwise we have to duplicate this setup all over the place.

See https://github.com/ManageIQ/manageiq-api-common/blob/master/lib/manageiq/api/common/logging.rb and https://github.com/ManageIQ/topological_inventory-api/blob/master/config/application.rb#L45